content/workshops/intro-to-css/_index.md

```diff
@@ -1,3 +1,27 @@
 ---
 title: Introduction to CSS
+ready: true
 ---
+
+## Design basics
+
+> There are two great days in a person's life
+>
+> - the day they are born and the day they discover why.
+
+The follow are to be demonstrated and addressed during the workshop
+
+## CSS foundation
+
+The followings are to be demonstrate/addressed during the workshop
+
+1. Inline CSS
+2. Internal CSS
+3. External CSS
+4. CSS Selectors (Element,Class,ID,Fonts & Colors)
+5. Box model
+6. Fonts and how to integrate them.
+
+### Resources
+
+1. [Learn CSS in 5 minutes - A tutorial for beginners](https://www.freecodecamp.org/news/get-started-with-css-in-5-minutes-e0804813fc3e/)
```

Comment:
This resource looks nice. It should probably be in `topics/intro-to-css` because  because:
- it introduces css
- it can be self-study material 

In this workshop you can have the following resource link:
```
{{% contentlink "topics/intro-to-css" %}}
```